extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/converter/type/QuantityTypeConverter.java

```diff
@@ -68,22 +71,25 @@ protected Object toBinding(QuantityType<? extends Quantity<?>> type, HmDatapoint
 
         // convert the given QuantityType to a QuantityType with the unit of the target datapoint
         switch (dp.getUnit()) {
-            case "minutes":
-                return type.toUnit(SmartHomeUnits.MINUTE);
-            case "day":
-                return type.toUnit(SmartHomeUnits.DAY);
-            case "month":
-                return type.toUnit(SmartHomeUnits.YEAR.divide(12));
-            case "year":
-                return type.toUnit(SmartHomeUnits.YEAR);
             case "Lux":
                 return type.toUnit(SmartHomeUnits.LUX);
             case "degree":
                 return type.toUnit(SmartHomeUnits.DEGREE_ANGLE);
+            case "100%":
+                return type.toUnit(SmartHomeUnits.ONE);
+            case "Â°C":
```

Comment:
Do you think a comment as to why this strange looking string needs to be handled would be useful here?

Reply:
I will add a string constant for the unit and add a comment to it.